{"type":"PR","pr":{"number":929,"head":"5b0797f5f7e8b89bad48ce9881471241aa2089ac"},"user":"alexeyvisyond","status":"draft","branch":"cms/error-root-cause-analysis","collection":"features","title":"Error Root Cause Analysis","description":"No Description!","objects":{"entry":{"path":"site/content/features/error-root-cause-analysis.md","sha":"29491cdebfc5b4afbf3dc0dc37cdc29b08136312"},"files":[{"path":"/site/static/source-img/track errors 1.png","sha":"27ca7e1ff7dd667d14e7ddee829d7cf8f96f86a1"},{"path":"/site/static/source-img/track errors 1.png","sha":"27ca7e1ff7dd667d14e7ddee829d7cf8f96f86a1"},{"path":"/site/static/source-img/track errors.png","sha":"597e6cdafff9a52d49d1a1c8a912d055ef8592ce"}]},"timeStamp":"2018-08-02T19:04:37.789Z"}